### PR TITLE
image converted page layout develop

### PR DESCRIPTION
**Pull request by yoouung**
- develop image converted page layout
- rename downloadBtn into downloadDocBtn as downloadImgBtn is generated
- update downloadBtn's parameter

### CODE REVIEW

File: dt-demo-fe/src/components/document/result/downloadBtn.js [hunk 1]
```diff
@@ -48,19 +58,31 @@ const DownloadBtn = ({ type, pdfRef, contents }) => {
...
+        } else if (type === "imgConverterReport") {
+            // TODO: pdf creation for image converter
         }
     });
 
     const text =
         type === "filterReport"
             ? "검출 보고서 다운로드"
-            : type === "converterReport"
+            : type === "converterReport" || type === "imgConverterReport"
```
Comment (doorooful) [inline dt-demo-fe/src/components/document/result/downloadBtn.js:69]: How about simply copy this file under `/components/image` not to be confused?
I think it'd be better to separate this from `document` dir

**Review by doorooful — COMMENTED** [review]
Can you update commit message if this PR includes code update about txt downloading from document?

**Review by doorooful — APPROVED** [review]
Thank you 🤍

**Comment by yoouung** [comment]
### converted page layout screenshot
image

**Pull request merged**